azure-pipelines/publishSamples.yml

```diff
@@ -0,0 +1,27 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+  branches:
+    include: 
+    - dev 
```

Comment:
We should run this on `master`

Reply:
> We should run this on `master`

I believe we want to trigger the pipeline to copy the files to the Staging blob on successful merges to `dev`; and copy the files to the Production blob on successful merges to `master`. Kindly clarify @MaggieKimani1